src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
enum class naming should be singular
```suggestion
public enum Color {
```